Library/Homebrew/shims/linux/super/cc

```diff
@@ -184,6 +184,11 @@ class Cmd
       val  = chuzzle($1) || enum.next
       path = canonical_path(val)
       args << "-L#{val}" if keep?(path) && @lset.add?(path)
+    when /^-R(.+)?/
+      # -R is not the correct way to pass rpaths to the linker
+      val  = chuzzle($1) || enum.next
+      path = canonical_path(val)
+      args << "-Wl,-R," if keep?(path) && @lset.add?(path)
```

Comment:
Thanks, I'll try this out. I couldn't find any documentation of either `-R` or `-rpath` but it makes sense that it's a linker flag rather than a `gcc` flag.